database/store.go

```diff
@@ -40,22 +45,17 @@ func NewStore() (Store, error){
 	if err != nil {
 		log.Printf("failed to init tables: %v", err)
 	}
-	return &store{ db: db }, nil
+	return &store{db: db}, nil
 }
 
-func NewDBConnection() (*sql.DB, error){
+func NewDBConnection() (*sql.DB, error) {
 
-	host 		:= os.Getenv("DATABASE_HOST")
-	portString	:= os.Getenv("DB_PORT")
-	dbUser     := os.Getenv("POSTGRES_USER")
+	// todo: optimize this approach, looks ugly
+	host := os.Getenv("DATABASE_HOST")
+	portString := os.Getenv("DB_PORT")
+	dbUser := os.Getenv("POSTGRES_USER")
 	dbPassword := os.Getenv("POSTGRES_PASSWORD")
-	dbName     := os.Getenv("POSTGRES_DB")
-
-	//host 		:= "localhost"
-	//portString	:= "5432"
-	//dbUser     := "root"
-	//dbPassword := "root"
-	//dbName     := "mydb"
+	dbName := os.Getenv("POSTGRES_DB")
```

Comment:
Seems dirty, needs to be fetched in better way !

Reply:
Configuration file could be a way